Doc/library/sqlite3.rst

```diff
@@ -236,8 +236,8 @@ Module functions and constants
    Registers a callable to convert a bytestring from the database into a custom
    Python type. The callable will be invoked for all database values that are of
    the type *typename*. Confer the parameter *detect_types* of the :func:`connect`
-   function for how the type detection works. Note that the case of *typename* and
-   the name of the type in your query must match!
```

Comment:
Please retain the "must match" wording, e.g.: "... must match (case-insensitively)!"